crow/setup3.py

```diff
@@ -18,15 +18,15 @@
 except:
   eigen_flags = ""
 include_dirs=['include/distributions','include/utilities','contrib/include']
-if eigen_flags.startswith("-I"):
+if eigen_flags.startswith(b"-I"):
```

Comment:
This `b` ensures that the string following is encoded in bytes and contains only ASCII characters, whereas python 3 defaults to unicode for strings.

I was going to ask why this was needed here, but after reading the [documentation](https://docs.python.org/2/library/subprocess.html#subprocess.check_output), I realize it is necessary since subprocess.check_output returns a byte string.  The remaining question is if we hit the except block and store an empty unicode `str` type into eigen_flags, does this line complain or evaluate to False as expected?